client/js/lounge.js

```diff
@@ -844,7 +844,7 @@ $(function() {
 
 		socket.emit("input", {
 			target: chat.data("id"),
-			text: "/whois " + name
+			text: "/whois " + name + " " + name
```

Comment:
@xPaw, @DanielOaks, if you look at the third commit of this PR, I actually moved that logic to the server through a new input file.
Reason behind that is that with the second commit alone, clicking on a nick and typing `/whois nick` were giving different outputs, and it confused me for a second. Now `whois nick` directly sends `/whois nick nick` (just like `/away` sends `/away <empty reason>`) and it's nicer for the user.
I'll squash commits 2 and 3 if you're ok with that (commit 3 cancels out commit 2, but I kept it for now just in case).

@PolarizedIons, done.